File: PracticasJava/Practica7/src/Aplicacion.java

public class Aplicacion {
	public static void main(String[] args) {
		JuegoAdivinaNumero j1= new JuegoAdivinaNumero(3, 0, 0);
		JuegoAdivinaNumero j2= new JuegoAdivinaImpar(2, 0, 0);
		JuegoAdivinaNumero j3= new JuegoAdivinaPar(3, 0, 0);
		j1.Juega();
		j2.Juega();
		j3.Juega();
	}
}

File: PracticasPython/Practica7/Aplicacion.py
from JuegoAdivinaNumero import JuegoAdivinaNumero  
from JuegoAdivinaImpar import JuegoAdivinaImpar    
from JuegoAdivinaPar import JuegoAdivinaPar        

class Aplicacion:
    @staticmethod
    def main():
        j1 = JuegoAdivinaNumero(1, 0, 3)
        j2 = JuegoAdivinaImpar(2, 0, 5) 
        j3 = JuegoAdivinaPar(2, 0, 2)  
        j1.Juega()
        j2.Juega()
        j3.Juega()
if __name__ == "__main__":
    Aplicacion.main()